web/libs/editor/src/components/App/ViewAll.tsx

```diff
@@ -0,0 +1,49 @@
+import clsx from "clsx";
+import { ff } from "@humansignal/core";
+import { usePersistentState } from "@humansignal/core/lib/hooks/usePersistentState";
+import type { MSTAnnotation, MSTStore } from "../../stores/types";
+import TaskSummary from "../TaskSummary/TaskSummary";
+import Grid from "./Grid";
+
+import styles from "./ViewAll.module.scss";
+
+type Props = {
+  store: MSTStore["annotationStore"];
+  annotations: MSTAnnotation[];
+  root: any;
+};
+
+const Tab = ({ title, active, onSelect }: { title: string; active: boolean; onSelect: () => void }) => {
+  return (
+    <div className={clsx(styles.tab, { [styles.active]: active })} onClick={onSelect}>
+      {title}
+    </div>
+  );
+};
+
+export const ViewAll = ({ store: annotationStore, annotations, root }: Props) => {
+  const [tab, setTab] = usePersistentState<"summary" | "compare">("view-all-tab", "summary");
```

Comment:
`"summary"` and `"compare"` could be defined as constants or something like that.